src/main/java/wolox/training/models/Book.java

```diff
@@ -90,6 +93,7 @@ public String getGenre() {
     }
 
     public void setGenre(String genre) {
+        Preconditions.checkNotNull(genre, "Genre can't be null");
```

Comment:
Genre can be null :smile: [book model card](https://trello.com/c/F1uZqeMs/14-crear-modelo-de-book)